test/data_types/cases_int/1.source_callbacks.c

```diff
@@ -2,7 +2,7 @@
 #include <stdint.h>
 
 void btx_push_usr_messages(void *btx_handle, void *usr_data, btx_source_status_t *status) {
-    btx_push_message_event(btx_handle, UINT64_MAX, UINT64_MAX, UINT32_MAX, INT64_MAX, INT64_MAX, INT32_MAX);
+    btx_push_message_event(btx_handle, 0, 0, 0, INT64_MIN, INT64_MIN, INT32_MIN);
```

Comment:
why 0, 0, 0?

Reply:
`stdint.h` does not define `UINT64_MIN` neither UINT32_MIN.

Reply:
Oh funny!